test/common/http/utility_test.cc

```diff
@@ -1289,6 +1289,10 @@ TEST(Url, ParsingTest) {
   // Test url with non-default ports.
   validateUrl("https://www.host.com:8443", "https", "www.host.com:8443", "/", 8443);
   validateUrl("http://www.host.com:8080", "http", "www.host.com:8080", "/", 8080);
+
+  // Test having unicode chars in the URL.
+  validateUrl("https://example.com/\xe5\x85\x89", "https", "example.com", "/%E5%85%89", 443);
+  validateUrl("https://example.com/光", "https", "example.com", "/%E5%85%89", 443);
```

Comment:
use `u8""` string literal.

Reply:
Hum, seems like adding "u8" fails the windows test. Where I can find the log for a particular test on CI? In this case: C:/_eb/execroot/envoy/bazel-out/x64_windows-opt/testlogs/test/common/http/utility_test/test.log?